tests/MongoDB.Bson.Tests/Specifications/bson-corpus/BsonCorpusTestRunner.cs

```diff
@@ -35,6 +35,33 @@ public void RunTestDefinition(JsonDrivenTestCase testCase)
         {
             var test = testCase.Test;
             var testType = test["type"].AsString;
+            var shared = testCase.Shared;
```

Comment:
`shared` is higher level than `test` so can you order the assignments like this:

    var shared = testCase.Shared;
    var test = testCase.Test;

Reply:
Also, move the assignment of the helper variable `testType` closer to where it is needed (see below).

Reply:
Done.